projects/go-lib/src/lib/components/go-table/go-table.component.html

```diff
@@ -1,13 +1,19 @@
 <div class="go-table-container" *ngIf="showTable">
+  <div class="go-table-toolbar go-table-toolbar--top" *ngIf="tableTitle">
+    <h5 class="go-heading--no-margin">{{ tableTitle }}</h5>
+  </div>
   <div class="go-table">
     <table class="go-table__table" *ngIf="hasData(); else noDataTable">
       <thead>
         <tr>
-          <th *ngFor="let col of columns" class="go-table__th" [ngClass]="columnClasses(col.field)" (click)="toggleSort(col.field)">
-            <ng-container *ngIf="!col.goTableHead; else columnHeaderOutlet">{{ col.title || col.field }}</ng-container>
-            <ng-template #columnHeaderOutlet>
-              <ng-container *ngTemplateOutlet="col.goTableHead"></ng-container>
-            </ng-template>
+          <th *ngFor="let col of columns" class="go-table__th" (click)="toggleSort(col.field)">
+            <div class="go-table__th__content">
+              <ng-container *ngIf="!col.goTableHead; else columnHeaderOutlet">{{ col.title || col.field }}</ng-container>
```

Comment:
This syntax for if statements is weird, but neat. 🤔

Reply:
Yeah, thought it was a little weird at first, but makes sense. 😬